examples/plugins/memcached.pp

```diff
@@ -1,7 +1,11 @@
 include ::collectd
 
 class { '::collectd::plugin::memcached':
-  host => '192.168.122.1',
-  port => '11211',
+  instances => {
+    'default' => {
+      'host' => '127.0.0.1',
```

Comment:
Is this a more sensible default that the old `192.168.122.1`? (I suppose it doesn't really matter.)